test/rubygems/test_gem_config_file.rb

```diff
@@ -465,21 +465,6 @@ def test_write_from_hash
     assert_equal %w[http://even-more-gems.example.com], Gem.sources
   end
 
-  def test_ignore_invalid_config_file
-    File.open @temp_conf, "w" do |fp|
-      fp.puts "invalid: yaml:"
-    end
-
-    begin
-      verbose = $VERBOSE
-      $VERBOSE = nil
-
-      util_config_file
-    ensure
-      $VERBOSE = verbose
-    end
-  end
```

Comment:
Same here, what happens now when loading an invalid file? It seems like previous behavior was ignoring it? Has that changed?